static/js/recent_topics_ui.js

```diff
@@ -445,6 +435,15 @@ function format_conversation(conversation_data) {
                 context.user_circle_class = buddy_data.get_user_circle_class(user_id);
             }
         }
+
+        // Display in most recent sender first order
+        all_senders = recent_senders.get_pm_recent_senders(context.user_ids_string);
```

Comment:
So there's a corner case to think about -- what to do with users who've never sent a message in the topic? I think there's two defensible options: Not show them at all, or show them with different styling. In either case, I think perhaps we should change `recent_senders.get_pm_recent_senders` to return two things (probably via an object), one being the recent senders in sorted order, and the other being everyone else. I think we can start with implementing "just show the users who actually participated" to match the UI.

We should make sure the tests cover this corner case too.

Reply:
Ok, updated!